docs/clusterdefinition.md

```diff
@@ -55,6 +55,7 @@ Here are the valid values for the orchestrator types:
 |vnetSubnetId|no|specifies the Id of an alternate VNET subnet.  The subnet id must specify a valid VNET ID owned by the same subscription. ([bring your own VNET examples](../examples/vnet))|
 |extensions|no|This is an array of extensions.  This indicates that the extension be run on a single master.  The name in the extensions array must exactly match the extension name in the extensionProfiles.|
 |vnetCidr|no| specifies the vnet cidr when using custom Vnets ([bring your own VNET examples](../examples/vnet))|
+|distro|no| Allows OS selection for Master nodes. Currently supported values are: `ubuntu` and `coreos`. If undefined, `ubuntu` is used as the OS type for Master and Agent nodes. Agent OS is determined by the Master node distro. Supported Orchestrators: Kubernetes |

```

Comment:
"Agent OS is determined by the Master node distro" @richardjortega does this mean that the agent profile distro is disregarded no matter what? I think it would be better to take the other approach we discussed: Master profile distro parameter determines master distro and agent profile distro determines agent distro. For both, empty field defaults to ubuntu. What do you think?

Reply:
Yes, discussed the details with Anthony yesterday. I think I pushed up a WIP and it kicked off a build. They will be independent but will have to do more testing. Also I have merged the files for custom data together.

Reply:
Great!